### PR TITLE
update BigPicture & BigTitle on all pages, add mobile style to BigTit…

### PR DESCRIPTION
**Pull request by Gonaline**
…le, use context for data players

### CODE REVIEW

**Review by FrancoisSalmon — APPROVED** [review]
ok

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

File: frontend/src/services/context/Ctx.jsx [hunk 1]
```diff
@@ -12,79 +12,79 @@ export function CtxProvider({ children }) {
       name: "FRECON-DEMOUGE Romane",
       role: "role-1",
       number: 1,
-      img: "./src/assets/pictures/cards/players/photo.png",
+      img: "./src/assets/images/cards/players/photo.png",
```
Comment (LoicBrassart) [inline frontend/src/services/context/Ctx.jsx:15]: If I'm not mistaken, you could replace with `/images/cards/players/photo.png`

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**